app/src/main/java/com/keylesspalace/tusky/util/AbsoluteTime.kt

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2017 Andrew Dawson
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.util
+
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Date
+import java.util.Locale
+
+public class AbsoluteTime {
+    companion object {
+        private val sameDaySdf = SimpleDateFormat("HH:mm", Locale.getDefault())
```

Comment:
Quoting the Android Studio warning: 
> Assigning Locale.getDefault() to a final static field will not work correctly if the user changes locale while the app is running

I'd create one `AbsoluteTime` instance for each `StatusBaseViewHolder`/`StatusViewHelper` it is used in to fix that.